src/newgrf.cpp

```diff
@@ -6589,6 +6589,31 @@ static void SkipIf(ByteReader *buf)
 				result = rt != INVALID_ROADTYPE && RoadTypeIsTram(rt);
 				break;
 			}
+			case 0x13: {
+				uint r1 = GB(cond_val, 0, 8);
+				uint r2 = GB(cond_val, 8, 8);
+				if (r1 >= _cur.grffile->railtype_list.size() || r2 >= _cur.grffile->railtype_list.size()) {
+					result = false;
```

Comment:
Similar places issue a grfmsg() in this case. Though not sure whether this case always triggers in GRF loading stages before reservation.

Reply:
i'll take suggestions on what debuglevel this should be shown.

as for earlier loading stages, i guess that applies to action 9 only? not sure if the check makes any sense there.